### PR TITLE
DS providers listview

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What
DS integration in providers list view

## Issue
State => draft
SettingsIcon svg's size in Th is forced to 0.35rem in DS (used VisuallyHidden until fixed)
Switch component in listview? (user has to enter client ID / secret before switching status state)
Row + Stack : need align items + margin for icon && name

## Demo
Capture d’écran 2021-08-18 à 14 51 02

### CODE REVIEW

**Comment by maevalienard** [comment]
By any chance, is it possible to add a Capital letter at the beginning of "Enable" & "Disable"? :)

**Comment by ronronscelestes** [comment]
> By any chance, is it possible to add a Capital letter at the beginning of "Enable" & "Disable"? :)

Yes I will do that and use translated messages everywhere, I forgot about it!

File: packages/plugins/users-permissions/admin/src/pages/Providers/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React, { useCallback, useMemo, useRef, useState } from 'react';
 import { useIntl } from 'react-intl';
-import { Header, List } from '@buffetjs/custom';
-import { Text } from '@buffetjs/core';
-import { Pencil } from '@buffetjs/icons';
+// import { Header, List } from '@buffetjs/custom';
```
Comment (soupette) [inline packages/plugins/users-permissions/admin/src/pages/Providers/index.js:3]: You can remove these lines

File: packages/plugins/users-permissions/admin/src/pages/Providers/index.js [hunk 2]
```diff
@@ -38,6 +49,7 @@ const ProvidersPage = () => {
   const updatePermissions = useMemo(() => {
     return { update: pluginPermissions.updateProviders };
   }, []);
+  console.log(updatePermissions);
```
Comment (soupette) [inline packages/plugins/users-permissions/admin/src/pages/Providers/index.js:52]: You can remove this also

File: packages/plugins/users-permissions/admin/src/pages/Providers/index.js [hunk 3]
```diff
@@ -46,16 +58,14 @@ const ProvidersPage = () => {
     dispatchSubmitSucceeded,
     formErrors,
     handleChange,
-    isLoading,
-    isLoadingForPermissions,
+    // isLoading,
```
Comment (soupette) [inline packages/plugins/users-permissions/admin/src/pages/Providers/index.js:61]: Can you add a loading state ?

File: packages/plugins/users-permissions/admin/src/pages/Providers/index.js [hunk 4]
```diff
@@ -194,42 +180,74 @@ const ProvidersPage = () => {
...
-                  {provider.enabled ? 'Enabled' : 'Disabled'}
-                </Text>
-              </td>
-            </ListRow>
-          )}
+      <Main labelledBy={formatMessage({ id: getTrad('HeaderNav.link.providers') })}>
+        <HeaderLayout
+          as="h1"
+          id="providers"
+          title={formatMessage({ id: getTrad('HeaderNav.link.providers') })}
```
Comment (soupette) [inline packages/plugins/users-permissions/admin/src/pages/Providers/index.js:189]: `formatMessage({ id: getTrad('HeaderNav.link.providers', defaultMessage: 'Providers' })`

File: packages/plugins/users-permissions/admin/src/pages/Providers/index.js [hunk 5]
```diff
@@ -194,42 +180,74 @@ const ProvidersPage = () => {
...
         />
-      </div>
+        <ContentLayout>
+          <Table colCount={4} rowCount={rowCount + 1}>
+            <Thead>
+              <Tr>
+                <Th>
+                  <TableLabel>
+                    <VisuallyHidden>
+                      {formatMessage({ id: getTrad('Providers.image') })}
```
Comment (soupette) [inline packages/plugins/users-permissions/admin/src/pages/Providers/index.js:198]: `{formatMessage({ id: getTrad('Providers.image') , defaultMessage: 'Image' })}`

File: packages/plugins/users-permissions/admin/src/pages/Providers/index.js [hunk 6]
```diff
@@ -194,42 +180,74 @@ const ProvidersPage = () => {
...
+                    </VisuallyHidden>
+                  </TableLabel>
+                </Th>
+                <Th>
+                  <TableLabel>{formatMessage({ id: getTrad('Providers.name') })}</TableLabel>
```
Comment (soupette) [inline packages/plugins/users-permissions/admin/src/pages/Providers/index.js:203]: Same for the others can you add a `defaultMessage` ?

**Review by soupette — CHANGES_REQUESTED** [review]
Can you also add some tests?

**Review by soupette — APPROVED** [review]
LGTM

**Pull request merged**